backend/app/tests/utils/utils.py

```
@@ -4,13 +4,15 @@
 
 import pytest
+from pydantic import EmailStr
 from fastapi.testclient import TestClient
 from sqlmodel import Session, select
 
 from app.core.config import settings
-from app.crud.user import get_user_by_email
-from app.crud.api_key import get_api_key_by_value
-from app.models import APIKeyPublic, Project
+from app.crud import get_user_by_email, get_api_key_by_user_id
+from app.models import APIKeyPublic, User, APIKey
```

Ruff (0.12.2)

`app.models.User` imported but unused

Remove unused import

(F401)